test/tests/ver-1fd/comparison_ver-1fd.py

```diff
@@ -0,0 +1,59 @@
+import csv
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import gridspec
+import pandas as pd
+from scipy import special
+import os
+import math
```

Comment:
```suggestion
import math
from scipy.special import erfc
from np import sqrt, exp
```

Reply:
Wouldn't `input numpy as np` and `from scipy import special` already pull in `erfc`, `sqrt, and `exp` respectively? If so, why be redundant?

Reply:
I removed `from np import sqrt, exp` and added `np.` before the functions `sqrt` and `exp`.

Reply:
I mostly prefer importing the specific functions because it makes the equation easier to read. Adding ``` np. ``` in each function in an equation just makes it unnecessarily long.

Reply:
I prefer seeing the implementation of the particular function being used inline where its used rather than at the top of the file, but I grant that that's a personal code preference. I also prefer being explicit about namespaces in C++ rather than utilizing the `using NameSpace::foo` syntax. 

@chaibhave does PEP8 have a guideline for this? If that is something we intend to stick closer to, we should consider that input.

Reply:
Crashing the party to add my 2¢. PEP8 doesn't really care either way, but I'd recommend the `np.` syntax as it's more common. The only time a `from` style numpy import has caused issues for me was when using `numpy` and `sympy` together or attempting to use reduction operators like `max` from `math` as they have different functional forms.